application_features/espresso/5.2.1.yml

```diff
@@ -0,0 +1,27 @@
+#  structure:
+#  directory:  application name
+#  file:    version
+#  1st level: executable
+#  2nd level: build
+#  3rd level: model node path
+#  4th level: model node data
```

Comment:
1. The "structure:" at the top is confusing b/c we assume yaml elsewhere
2. This we can move to README